exercises/java-collections/src/main/java/com/cbfacademy/CollectionsExercises.java

```diff
@@ -4,26 +4,80 @@
 
 public class CollectionsExercises {
 
-    public LinkedList<Integer> useLinkedList() {
+
+
+    public static LinkedList<Integer> useLinkedList() {
         // TODO: create an empty linked list
         //  - add 4 as the first element of the list
         //  - then add 5, 6, 8, 2, 9 to the the list
         //  - add another 2 as the last element of the list
         //  - add 4 as the 3rd element of the list
         //  - invoke the method element() on the list and print the result on the screen
         //  - return the list
-        throw new RuntimeException("Not implemented");
+
+        LinkedList<Integer> newList = new LinkedList<>();
+        newList.addFirst(4); // 0
+        newList.add(5);
+        newList.add(6);
+        newList.add(8);
+        newList.add(2);
+        newList.add(9); 
+        newList.addLast(2);
+        newList.add(2, 4);
+
+        int element = newList.element();
+
+        System.out.println(element);
```

Comment:
This variable on line 28 is redundant as it's only being used on line 30. You could instead write this as 
```
System.out.println(newList.element());
```